src/lib/library/library.factory.ts

```diff
@@ -43,6 +44,20 @@ export function main(options: LibraryOptions): Rule {
   ]);
 }
 
+function getDefaultLibraryPrefix(defaultLibraryPrefix = '@app') {
+  try {
+    const nestCliJson = JSON.parse(
+      readFileSync('./nest-cli.json', 'utf-8'),
```

Comment:
at `@netjs/cli` repo we have this code to read the configuration file: https://github.com/nestjs/nest-cli/blob/d7c855ad3d8493bc5e43ebcb3199aab75232d7a4/lib/configuration/nest-configuration.loader.ts#L17 
![image](https://github.com/nestjs/schematics/assets/13461315/60872e66-f56f-419d-9015-f937274fce24)

I'm not sure if we should do the same here...

Reply:
I know that. There are a few json files.
But latest version of nestjs generates 'nest-cli.json' file. doesn't it?
This feature that I suggested is new, so I think supporting only 'nest-cli.json' is ok.
But becuase @micalevisk 's suggestion is very good idea and It is not that hard to fix, I don't care.
which is better?

Reply:
We should use the NestConfigurationLoader here

Reply:
@kamilmysliwiec 
Thank you for your advice.
I have investigated whole code in schematics repo, but schematics project only care about these two files, 'nest-cli.json', 'nest.json'.
Only nest-cli project has NestConfigurationLoader. and it's FileSystemReader class read some json files.
Well, I just pushed new code that is checking both nest-cli.json and nest.json to get defaultLibraryPrefix.
If you want something else, give me direction in more detail. I'm ready to willingly apply that.

Reply:
Could you add `FileSystemReader` class to this package (https://github.dev/nestjs/nest-cli/blob/ad212b1b4c6a7777f232ab6c93f83a5fa0bc5fac/lib/readers/file-system.reader.ts#L5) and use it instead?